config/app.php

```diff
@@ -67,7 +67,7 @@
     |
     */
 
-    'timezone' => 'America/Toronto',
+    'timezone' => env('APP_TIMEZONE', 'UTC'),
```

Comment:
smart, GC. Remind me to update prod with this env var plz

Reply:
technically the app should work fine in UTC even with EST browsers. At least all my local tests are passing.